testplan/runnable/interactive/http.py

```diff
@@ -607,7 +609,9 @@ def post(self):
                         try:
                             report = copy.deepcopy(ihandler.report)
                             report.reset_uid()
-                            export_path = exporter.export(report)
+                            export_path = exporter.export(
```

Comment:
Now this export path will be a dictionary, it used to be a string. I think this need a bit oof thought, what to show to interactive users whne they export

Reply:
Also this seems does not use the helper that handles deprecated exporters